src/main/java/org/testng/TestNG.java

```diff
@@ -474,13 +474,17 @@ public void setThreadCount(int threadCount) {
   @Deprecated
   public void setParallel(String parallel) {
     if (parallel == null) {
-      setParallel(XmlSuite.ParallelMode.FALSE);
+      setParallel(XmlSuite.ParallelMode.NONE);
     } else {
       setParallel(XmlSuite.ParallelMode.getValidParallel(parallel));
     }
   }
 
   public void setParallel(XmlSuite.ParallelMode parallel) {
+    if (parallel == XmlSuite.ParallelMode.FALSE || parallel == XmlSuite.ParallelMode.TRUE) {
+      Utils.log("TestNG", 1, "[WARN] 'parallel' value 'true' and 'false' are deprecated, use 'none' instead.");
```

Comment:
No, I mean the warning is saying that two values are deprecated, it should indicate what to use instead.

Maybe two separate warnings: if the user specified `false`, tell them it's deprecated and to use `none` instead, and if they used `true`, tell them it's deprecated and use... what's the default again, `methods`?